### PR TITLE
Minor additions and corrections

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/hard-good-interval-longer-than-good-easy.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 When a card is in learning:
 
 - Again will be the first learning step
-- Hard will be the current learning step
+- Hard will be the average of Again and Good
```
Comment (dae) [inline src/hard-good-interval-longer-than-good-easy.md:6]:
```suggestion
- Hard will repeat the current learning step (except on the first step, where it is the average of first and second steps)```

File: src/what-spaced-repetition-algorithm.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Moves the card to the [next step](https://docs.ankiweb.net/deck-options.html?#le
 If the card was on the final step, the card is converted into a 
 review card (it 'graduates').
 
-- Easy 
+- Easy
```
Comment (dae) [inline src/what-spaced-repetition-algorithm.md:62]:
```suggestion
- Easy
```

File: src/what-spaced-repetition-algorithm.md [hunk 3]
```diff
@@ -108,15 +108,6 @@ increased beyond the value of *maximum interval*. Finally, all new
...
 
-After you select an ease button, Anki also applies a small amount of
-random “fuzz” to prevent cards that were introduced at the same time and
-given the same ratings from sticking together and always coming up for
-review on the same day. This fuzz appear on the interval
-buttons from 2.1.45+, but not in previous versions, so if you are using 
-a previous version and you’re 
-noticing a slight discrepancy between what you
-select and the intervals your cards actually get, this is probably the
-cause.
```
Comment (dae) [inline src/what-spaced-repetition-algorithm.md:119]: This applies to the v1/v2 scheduler, not a specific Anki version. Given that not everyone is using the v3 scheduler yet, perhaps it is worth keeping in?

**Review by dae — COMMENTED** [review]
Thanks cqg! A few comments/suggestions below.

File: src/what-spaced-repetition-algorithm.md (hunk 3 above)
Comment (dae) [reply src/what-spaced-repetition-algorithm.md:119]: Ah, just saw you moved this into the manual. Would it be worth a link at least?

Comment (cqg2) [reply src/what-spaced-repetition-algorithm.md:119]: To me, this section makes more sense in the main documentation, if we move it there, I don't really see any reason why we should add a link to that section here. 
Of course, that's just me :)

**Comment by cqg2** [comment]
Thanks again, Dae!

**Pull request merged**